aws/s3/lock.go

```diff
@@ -10,6 +10,28 @@ type Lock struct {
 	UUID string `json:"uuid,omitempty"`
 }
 
+type UserLock struct {
```

Comment:
what happens here if someone uploads a non-json file to this, and what happens if the it is json but doenst look like this format?

Reply:
Currently, we will just return the error when unmarshaling the object in `GetStruct`. However, the logic to upload/remove the lock object will live in the client. Users would only have permissions to upload objects to dev for testing, so there should never be a case were the lock file doesn't look like that format in the corporate deployers?